app/src/main/kotlin/cz/covid19cz/erouska/ui/login/LoginVM.kt

```diff
@@ -111,6 +130,7 @@ class LoginVM(
     }
 
     fun backPressed() {
+        smsCountDownTimer.cancel()
```

Comment:
Je to potřeba zcancellovat taky když uživatel odnaviguje z této obrazovky - onDestroyView?